### ISSUE TITLE
inconsistency of LoadImage

### ISSUE THREAD

**Issue by JianJuly** [Project-MONAI/MONAI#2902]
I have a nrrd file, of shape `256*256*42`

when i run
```python
from monai.transforms import LoadImage
img, metadata = LoadImage(reader='itkreader')(path)
print(metadata['spacing'])
print(metadata['spatial_shape'])
```
I got `array([1.3671875 , 1.3671875 , 5.20000458])` and `array([ 42, 256, 256])`

yet if i run
```python
import SimpleITK as sitk
img = sitk.ReadImage(path)
print(img.GetSpacing())
print(img.GetSize())
```
i got `(1.3671875, 1.3671875, 5.200004577636718)` and `(256, 256, 42)`

obviously, the values are inconsistency  between `metadata['spacing']` and `metadata['spatial_shape']`

**Comment by Nic-Ma** [comment]
Hi @JianJuly ,

Thanks for your interesting experiments.
To be consistent with Nibabel and other packages, we use `HWD` order in MONAI and updated the `affine` and `spatial_shape` metadata only, so the `spacing` meta is not correct, let me try to also update it soon.
@wyli @ericspod Do you guys think we should also update any other "dim related" meta info?

Thanks in advance.

**Comment by Nic-Ma** [comment]
Hi @JianJuly ,

And BTW, may I know your MONAI version?
We changed the ITKReader logic in v0.6, I need to double confirm whether it's really wrong.

Thanks.

**Comment by JianJuly** [comment]
@Nic-Ma Thank you. My monai version is 0.6.0.

**Comment by JianJuly** [comment]
@Nic-Ma Another question is, how can i use spacing in this case? I want to set its spacing from `(1.3671875, 1.3671875, 5.200004577636718)`to `(1, 1, 5)`

**Comment by wyli** [comment]
I think this is fixed in #2613, and the fix will be released in v0.7.
